rootfs/etc/services.d/pulseaudio/run

```diff
@@ -5,4 +5,11 @@
 export PULSE_STATE_PATH="/data/states"
 export LD_PRELOAD="/usr/local/lib/libjemalloc.so.2"
 
-exec pulseaudio --system -vvv --disallow-exit --exit-idle-time=-1 --disable-shm
+debug="$(jq '.debug' /config/plugin_audio.json)"
+log_level=""
+
+if [ "$debug" == 'true']; then
```

Comment:
```suggestion
if [ "$debug" = "true" ]; then
```

Space before `]` seems to be required. Single `=` is more POSIX. Quotes probably doesn't matter, but is nicer IMHO.